packages/rainbowkit/src/wallets/getDefaultWallets.ts

```diff
@@ -5,6 +5,7 @@ import { braveWallet } from './walletConnectors/braveWallet/braveWallet';
 import { coinbaseWallet } from './walletConnectors/coinbaseWallet/coinbaseWallet';
 import { injectedWallet } from './walletConnectors/injectedWallet/injectedWallet';
 import { metaMaskWallet } from './walletConnectors/metaMaskWallet/metaMaskWallet';
+import { rabbyWallet } from './walletConnectors/rabbyWallet/rabbyWallet';
```

Comment:
Please revert these changes. We'll have a better solution soon for a community-driven list of wallets.

Reply:
What do I need to do now, I haven't seen a new solution yet.

Reply:
@heisenberg-2077 Revert here and add to `packages/example` custom wallet list for testing